File: beginner/java/2630.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        int n, i, a, b, c;
        String val;

        n = scanner.nextInt();

        for(i = 1; i <= n; i++){
            val = scanner.next();
            
            a = scanner.nextInt();
            b = scanner.nextInt();
            c = scanner.nextInt();

            if(val.equals("eye")){
                System.out.printf("Caso #%d: %d%n", i, (int) (0.30*a + 0.59*b + 0.11*c));
            } else if(val.equals("mean")){
                System.out.printf("Caso #%d: %d%n", i, ((a + b + c) / 3));
            } else if(val.equals("max")){
                if(a > b && a > c)
                    System.out.printf("Caso #%d: %d%n", i, a);
                else if(b > a && b > c)
                    System.out.printf("Caso #%d: %d%n", i, b);
                else
                    System.out.printf("Caso #%d: %d%n", i, c);
            } else if(val.equals("min")){
                if(a < b && a < c)
                    System.out.printf("Caso #%d: %d%n", i, a);
                else if(b < a && b < c)
                    System.out.printf("Caso #%d: %d%n", i, b);
                else
                    System.out.printf("Caso #%d: %d%n", i, c);
            }
        }

        scanner.close();
    }
}

File: beginner/c/2630.c
#include <stdio.h>
#include <string.h>

int main() {
    
    int n, i, a, b, c;
    char val[50];

    scanf("%d", &n);

    for(i = 1; i <= n; i++){
        scanf("%s", val);
        scanf("%d %d %d", &a, &b, &c);

        if(strcmp(val,"eye") == 0){
            printf("Caso #%d: %d\n", i, (int) (0.30*a + 0.59*b + 0.11*c));
        } else if(strcmp(val,"mean") == 0){
            printf("Caso #%d: %d\n", i, ((a + b + c) / 3));
        } else if(strcmp(val,"max") == 0){
            if(a > b && a > c)
                printf("Caso #%d: %d\n", i, a);
            else if(b > a && b > c)
                printf("Caso #%d: %d\n", i, b);
            else
                printf("Caso #%d: %d\n", i, c);
        } else if(strcmp(val,"min") == 0){
            if(a < b && a < c)
                printf("Caso #%d: %d\n", i, a);
            else if(b < a && b < c)
                printf("Caso #%d: %d\n", i, b);
            else
                printf("Caso #%d: %d\n", i, c);
        }
    }

    return 0;
}
